sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ConnectionStringProperties.java

```diff
@@ -161,15 +159,16 @@ public String getSharedAccessSignature() {
      * The function checks for pre existing scheme of "sb://" , "http://" or "https://". If the scheme is not provided
      * in endpoint, it will set the default scheme to "sb://".
      */
-    private String validateAndUpdateDefaultScheme(final String endpoint, final String connectionString) {
-        String updatedEndpoint = endpoint.trim();
+    private String validateAndUpdateDefaultScheme(final String endpoint) {
 
         if (CoreUtils.isNullOrEmpty(endpoint)) {
-            throw logger.logExceptionAsError(new IllegalArgumentException(String.format(Locale.US,
-                ERROR_MESSAGE_ENDPOINT_FORMAT, connectionString)));
-
+            throw logger.logExceptionAsError(new IllegalArgumentException(
+                "'Endpoint' must be provided in 'connectionString'."));
         }
-        final String endpointLowerCase = endpoint.toLowerCase(Locale.getDefault());
+
+        String updatedEndpoint = endpoint.trim();
```

Comment:
Is there a reason not to chain this?

Also, not to use `Locale.getDefault()`. These endpoints are of invariant culture. It's possible that the user's machine has another locale.